.prettierrc.json

```
@@ -0,0 +1,11 @@
+{
+  "printWidth": 100,
```

Something that annoys me for way to long 😆 
```suggestion
  "printWidth": 80,
```